src/extensions/toolbar.ts

```diff
@@ -16,12 +17,20 @@ const LOCALSTORAGE_KEY = '_postHogToolbarParams'
 export class Toolbar {
     instance: PostHog
 
-    private _toolbarScriptLoaded = false
-
     constructor(instance: PostHog) {
         this.instance = instance
     }
 
+    // NOTE: We store the state of the toolbar in the global scope to avoid multiple instances of the SDK loading the toolbar
+    private setToolbarState(state: number) {
+        assignableWindow['ph_toolbar_state'] = state
+    }
+
+    // 0 = uninitiated, 1 = loading script, 2 = script loaded
```

Comment:
nitpicking... if these were constants in the file it still bundles nicely and stores a small value but the future traveller will have a better time